.github/workflows/gh-linux_mac.yml

```diff
@@ -0,0 +1,360 @@
+          make -C tiledb install
+
+          #- run: |
+          cd $GITHUB_WORKSPACE/build
+          ls -la
+
+          if [[ ( ${{ runner.os }} == "Linux" && "${{ matrix.TILEDB_S3 }}" == "ON" ) ]]; then
+            # make sure docker is still running...
+            printenv
+            docker ps -a
+          fi
+
+          make -j4 -C tiledb tiledb_unit
+
+          if [[ "${{ matrix.TILEDB_CI_ASAN }}" == "ON" ]]; then
+            export ASAN_OPTIONS=detect_leaks=0 LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libasan.so.4
+          fi
+
+          if [[ ( "${{ runner.os }}" == "macOS" && "${{ matrix.TILEDB_GCS }}" == "ON" )]]; then
+            # GCS unit tests are temporarily unsupported on CI. Fake success with
+            # this echo.
+            echo ::set-output name=TILEDB_CI_SUCCESS::1
+          else
+            # Bypass Catch2 Framework stdout interception with awk on test output
+            make check | awk '/1: ::set-output/{sub(/.*1: /, ""); print; next} 1'
+          fi
+          # Kill the running Minio server, OSX only because Linux runs it within
+          # docker.
+          if [[ ( "${{ runner.os }}" == "macOS" && "${{ matrix.TILEDB_S3 }}" == "ON" ) ]]; then
+            kill -n 9 $MINIO_PID
+          fi
+
+          # Kill the running Azurite server
+          if [[ "${{ matrix.TILEDB_AZURE }}" == "ON" ]]; then
+            kill -n 9 $AZURITE_PID
+          fi
+
+          # Kill the running GCS emulator server Linux only because OSX does not
+          # run the emulator
+          if [[ ( "${{ runner.os }}" == "macOS" && "${{ matrix.TILEDB_GCS }}" == "ON" ) ]]; then
+            kill -9 $GCS_PID
+          fi
+
+#          # - bash: |
+#          if [[ "$TILEDB_GCS" != "ON" ]]; then
+#            pushd $BUILD_REPOSITORY_LOCALPATH/examples/cmake_project
+#            mkdir build && cd build
+#            cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist .. && make
+#            ./ExampleExe
+#            if [[ "TILEDB_STATIC" == "ON" ]]; then
+#              ./ExampleExe_static
+#            fi
+#            popd
+#            # Build and run the PNG ingestion example.
+#            if [[ "$AGENT_OS" == "Linux" ]]; then
+#              # libpng (example dependency)
+#              sudo apt-get install libpng-dev
+#
+#              pushd $BUILD_REPOSITORY_LOCALPATH/examples/png_ingestion;
+#              mkdir build && cd build;
+#              cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist .. && make;
+#              ./tiledb_png $BUILD_REPOSITORY_LOCALPATH/doc/source/figures/Scarlet-Macaw-small.png /tmp/pngarray output.png;
+#              popd;
+#            fi
+#          fi
+#
+#          # Build the benchmarks (but do not run them).
+#          pushd $BUILD_REPOSITORY_LOCALPATH/test/benchmarking && \
+#            mkdir build && cd build && \
+#            cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist ../src && make && \
+#          popd
+#
+#          testfile=$(mktemp)
```

Comment:
I believe this was a test of direct linkage. May not be necessary, or could be a follow-up.